src/Campaigns/Repositories/CampaignRepository.php

```diff
@@ -42,11 +44,18 @@ public function getById(int $id)
      *
      * @throws Exception|InvalidArgumentException
      */
-    public function insert(Campaign $campaign): void
+    public function insert(Campaign $campaign, DonationForm $defaultDonationForm = null): void
```

Comment:
Why do we need this? Creating a new campaign should automatically create the default campaign form. We are not creating campaigns when creating new forms, you won't even be able to create a new form without creating a campaign first.

Reply:
Resolved in 0714b47594199a3bb112162fbf5e9bfcb0e8d6f4